app/controllers/admin/paypal_preferences_controller.rb

```diff
@@ -58,6 +63,9 @@ def index
       admin_getting_started_guide_path,
       Admin::OnboardingWizard.new(@current_community.id).setup_status)
 
+    currencies = MarketplaceService::AvailableCurrencies::CURRENCIES
+    available_currencies = [currency].concat(currencies.select { |c| c != currency })
```

Comment:
In this case, we can already rely on the fact that the user is using PayPal, right? So, could we also rely that the current currency is in the list of available currencies, without adding it to the list?